src/core/plugins/core-commands/server/commands/moderator/vehicle.ts

```diff
@@ -1,4 +1,5 @@
 import alt from 'alt-server';
+import { isPedOnFoot } from 'natives';
```

Comment:
This wont work because you can't import natives on Serverside. This will cause an Error

Reply:
Just seen it lol - this have to be removed

Reply:
Removed it, my mistake.